rpa-manager/modules/ROOT/pages/usermanagement-manage.adoc

```diff
@@ -130,6 +152,16 @@ Both the computer on which the RPA Manager is installed, and the mobile device o
 . Click *Enable Two-Factor Authentication*.
 . Log in using your password and 2FA.
 
+== Edit a User
+
+Edit a user to change his data or permissions.
```

Comment:
We don't use gender-specific words such as "his." Make the text plural to get around this.